### PR TITLE
A few revisions & additions

### PR DESCRIPTION
**Pull request by derek**
1) Added Events section and Delegation examples (Issue #9)
2) Added JSON & JSONP Ajax examples
3) Revised 'Getting Started' section - I felt the previous section didn't give any newbies to either library a very good understanding of implementation differences between the two.  Starting with the basics of how to include & use each library should ease the curve.

Please review for accuracy.  Oh, and as we learned from last pull, typos too. :)

### CODE REVIEW

**Comment by lsmith** [comment]
In the jsonp commit, Y.jsonp("service.php?callback={callback}", successFn); not {callbackFunc}, and you can do var service = new Y.JSONPRequest(url, { allowCache: true, on: { success: successFn } }); service.send(); service.send(); to reuse the same generated callback name.  Not sure if that's worth mentioning.

File: index.html [hunk 1]
```diff
@@ -457,6 +411,118 @@ Y.JSON.stringify({name: "Douglas"});
...
+Y.one("#foo").on({
+  click    : <i>fn</i>,
+  focus    : <i>fn</i>,
+  blur     : <i>fn</i>,
+  mouseout : <i>fn</i>,
+  mouseover: <i>fn</i>
+})
+
+<i>// Or attach a single subscriber to multiple events.</i>
+Y.one("#foo").on(['click', 'focus', 'blur', <i>fn</i>)
```
Comment (rgrove) [inline index.html]: Missing a closing `]` in the array literal here.

File: index.html [hunk 2]
```diff
@@ -457,6 +411,118 @@ Y.JSON.stringify({name: "Douglas"});
...
+
+    <tr>
+      <td class="code">
+<pre class="code jq">
+$('#target').live('click', <i>fn</i>);
+</pre>
+      </td>
+      <td class="code">
+<pre class="code yui">
+Y.one(document).delegate('click', <i>fn</i>, '#target');
```
Comment (rgrove) [inline index.html]: YUI best practice is to use `Y.config.doc` rather than referring to `document` directly, for better compat with server-side JS, Caja, etc.

File: index.html [hunk 3]
```diff
@@ -457,6 +411,118 @@ Y.JSON.stringify({name: "Douglas"});
...
+
+    <tr>
+      <td class="code">
+<pre class="code jq">
+$('#container').delegate('#target', 'click', <i>fn</i>);
+</pre>
+      </td>
+      <td class="code">
+<pre class="code yui">
+Y.one("#container").delegate('click', <i>fn</i>, '#target');
```
Comment (rgrove) [inline index.html]: Nitpick: stick to single quotes for consistency (also see "#foo" below).

**Comment by rgrove** [comment]
Apart from those inline comments, this looks good to me. Thanks Derek!

Handing off to @paulirish for jQuery review.

File: index.html [hunk 4]
```diff
@@ -33,45 +34,57 @@
...
-        <pre class="code jq">$<i>.foo.bar()</i></pre>
+<pre class="code jq">
+&lt;script src="https://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"&gt;&lt;/script&gt;
+&lt;script src="https://somedomain.com/path/to/plugin.js"&gt;&lt;/script&gt;
+&lt;script src="https://somedomain.com/path/to/anotherplugin.js"&gt;&lt;/script&gt;
+
+&lt;script&gt;
+  $(document).ready(function() {
+    // Do stuff with the $ object here
+    $<i>.foo.bar()</i>
```
Comment (paulirish) [inline index.html:56]:
```
$(elem)<i>foo()</i>
```

is more like it.

**Comment by paulirish** [comment]
added one correction. rest seems aiight.

**Comment by rgrove** [comment]
@derek, did you see Paul's correction above? Also, any chance you could resolve conflicts with latest master? This request has been open for a while and is no longer easily mergeable.

**Comment by rgrove** [comment]
I assume this is dead.
